com.unity.render-pipelines.universal/Editor/ShaderGUI/BaseShaderGUI.cs

```diff
@@ -214,20 +254,55 @@ public void ShaderPropertiesGUI(Material material)
         ////////////////////////////////////
         #region DrawingFunctions
 
+        public void DrawShaderGraphProperties(Material material, IEnumerable<MaterialProperty> properties)
```

Comment:
Does this function need to be public? If so we need public API docs. We have currently some missing API docs in this class and task to fix it, but we should not introduce more public undocumented API at least.

Reply:
Nothing "needs" to be public for it to work.   I guess the question is whether the public would like to be able to use it.   In this case, ShaderGraph is exposing a function to effectively replace this one, so I can make this one internal for now.